conceptual/Npgsql/index.md

```diff
@@ -18,21 +18,21 @@ var connString = "Host=myserver;Username=mylogin;Password=mypass;Database=mydata
 
 using (var conn = new NpgsqlConnection(connString))
```

Comment:
Another `await` here maybe? (:

Reply:
Yeah, you're right (even if it doesn't make any difference for the common pooled usage).